crates/re_log_types/src/msg_bundle.rs

```diff
@@ -418,18 +225,17 @@ impl std::fmt::Display for MsgBundle {
     }
 }
 
-/// Pack the passed iterator of `ComponentBundle` into a `(Schema, StructArray)` tuple.
+/// Pack the passed iterator of `DataCell` into a `(Schema, StructArray)` tuple.
```

Comment:
```suggestion
/// Pack the passed iterator of [`DataCell`] into a `(Schema, StructArray)` tuple.
```

🤷 

makes the code more robust against future refactors/renames